packages/apollos-data-prayer/src/prayer-requests/resolver.js

```diff
@@ -5,13 +5,13 @@ export default {
   Query: {
     prayers: (root, args, { dataSources }) =>
       dataSources.PrayerRequest.getAll(),
-    campusPrayers: (root, { campusId }, { dataSources }) =>
-      dataSources.PrayerRequest.getAllByCampus(campusId),
+    campusPrayers: (root, args, { dataSources }) =>
```

Comment:
The actual function - `getAllByCampus` still accepts an `id` parameter. I guess that would be so that if we wanted to use that function someplace else and still pass a campus, we could?

Reply:
yes